GoogleForCreators/web-stories-wp
# Template Updates: Fashion

## Context

Part of 1.7 release is updating existing templates to avoid custom animations and unnecessary prepublish checklist suggestions.

## Summary

Fashion Template revisions
- remove custom animations and replace them with effects available in editor
- add starbursts and arrows as stickers 
- adjust copy of page 7
- Update red/orange color of text 

## Relevant Technical Choices

Used my own judgement for what direction everything should fly in.

## To-do

If the goal of updating the red font was to make it pass the PPC recommendation, it's still not - so we might need to adjust that again depending on design feedback.


## User-facing changes

See figma for revised template: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7357%3A546501
List of changes can be found here: #6973 

**Before**

https://user-images.githubusercontent.com/10720454/114231549-73307400-992f-11eb-85e3-389edf0b585d.mp4

**After**

https://user-images.githubusercontent.com/10720454/114231526-6ca1fc80-992f-11eb-81ea-b8cdde71618b.mp4



## Testing Instructions

Use the fashion template from dashboard/explore templates and see that the template has been updated in line with Figma. There should be no more custom animations.

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6973
